composer.json

```diff
@@ -11,12 +11,23 @@
 		"files": [ "wpcli-vulnerability-scanner.php" ]
 	},
 	"require-dev": {
-		"behat/behat": "~2.5"
+		"wp-cli/wp-cli-tests": "^3.1",
+		"behat/behat": "^3.11"
 	},
 	"authors": [
 		{
 			"name": "10up",
 			"homepage": "https://10up.com"
 		}
-	]
+	],
+	"config": {
+		"allow-plugins": {
+			"dealerdirect/phpcodesniffer-composer-installer": true
+		},
+		"process-timeout": 1800
+	},
+	"scripts": {
+        "behat": "run-behat-tests",
+        "prepare-tests": "install-package-tests"
+    }
```

Comment:
Looks like some spacing off here

Reply:
@dkotter mind applying updates for this so I can merge in / release by end of week?

Reply:
Done